web-wallet/src/lib/contracts/executeSend.js

```diff
@@ -3,16 +3,17 @@ import { Gas } from "$lib/vendor/w3sper.js/src/mod";
 
 import { walletStore } from "$lib/stores";
 
-/** @type {(to: string, amount: bigint, gasPrice: bigint, gasLimit: bigint) => Promise<string>} */
-const executeSend = (to, amount, gasPrice, gasLimit) => {
+/** @type {(to: string, amount: bigint, gasPrice: bigint, gasLimit: bigint, memo: any) => Promise<string>} */
```

Comment:
Shouldn't be string the type for "memo"?

(see above about the arguments order)